packages/vite/rollup.config.js

```diff
@@ -123,8 +123,12 @@ const nodeConfig = {
         replacement: `eval('require')('sugarss')`
       },
       'import-fresh/index.js': {
-        src: 'require(filePath)',
+        pattern: /require\(filePath\)/g,
```

Comment:
This change was unnecessary and caused https://github.com/vitejs/vite/issues/1306